app/cdap/api/settings.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2024 Cask Data, Inc.
```

Comment:
nit: 2025 in all new files

Reply:
Yeah, will update the new files. Thanks for pointing out !